resources/metadata/kubernetesDeploy.yaml

```diff
@@ -236,6 +236,30 @@ spec:
           - PARAMETERS
           - STAGES
           - STEPS
+      - name: imageNames
+        aliases:
+          - name: deployImage
+        type: "[]string"
+        description: List of names of the images to be deployed.
+        resourceRef:
+          - name: commonPipelineEnvironment
+            param: container/imageNames
+        scope:
+          - PARAMETERS
+          - STAGES
+          - STEPS
+      - name: imageNameTags
+        aliases:
+          - name: deployImage
```

Comment:
should we use different aliases for the `imageNames` and `imageNameTags` properties, if we need them at all?

Reply:
Copy&Paste error, will remove them, thanks